odxtools/dataobjectproperty.py

```diff
@@ -136,11 +136,18 @@ def decode_from_pdu(self, decode_state: DecodeState) -> ParameterValue:
 
         if self.compu_method.is_valid_internal_value(internal):
             return self.compu_method.convert_internal_to_physical(internal)
-        else:
-            # TODO: How to prevent this?
-            raise DecodeError(
-                f"DOP {self.short_name} could not convert the coded value "
-                f" {repr(internal)} to physical type {self.physical_type.base_data_type}.")
+
+        internal_to_phys = self.compu_method.compu_internal_to_phys
+        default_value = internal_to_phys.compu_default_value if internal_to_phys else None
+
+        if default_value is not None:
+            return str(default_value)
```

Comment:
do you have a reference to the spec for this? It still seems fishy to me to fall back to a default value when receiving an undecodable value. (encoding a PDU where the parameter in question is left unspecified is a different story IMO.)

Reply:
ok, thanks. That's a build-in footgun IMO, but...